composer.lock

```diff
@@ -2021,16 +2082,16 @@
         },
         {
             "name": "mccool/laravel-auto-presenter",
-            "version": "4.2.1",
+            "version": "v4.2.1",
```

Comment:
Not really. It was my fault. I messed the name and retagged, but composer doesn't bother with that and uses it's stale cache.